src/models/playbook/PlaybookModel.js

```diff
@@ -1,7 +1,7 @@
 /**
  * @requires Services
  */
-import * as TextIndentService from '../../services/utils/TextIndentService';
+import * as TextService from '../../services/utils/TextService';
```

Comment:
more of a question - is this the syntax we need to use. 
eg. does `import {TextServiceThingy} from '../../services/utils/TextService';` not work ?

Reply:
I think import {} will work just fine, i just pull it all in so it feels like a class when its used

Reply:
ah gotcha - yeah cool good idea. I like that better